Add deRSE23 submission abstract (#24)
fortran-lang/talks: deRSE23/README.md

# deRSE23 Submission

Abstract (and slides) for the [deRSE23](https://de-rse23.sciencesconf.org/) conference.

* Title: Introducing the Fortran-lang community
* Presenter: Ivan Pribec

> * Co-authors: Vincent Magnin, Asdrubal Lozada-Blanco, Ondrej Certik, Jeremie Vandenplas, Milan Curcic

Reviewer: ```suggestion
* Co-authors: Vincent Magnin, Asdrubal Lozada-Blanco, Ondrej Certik, Jeremie Vandenplas, Milan Curcic, Rohit Goswami
```

## Abstract

> A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering, and evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. To address this, an open-source community Fortran-lang has formed in 2019. Here, we present the efforts of Fortran-lang such as the Fortran package manager, standard library, website, LFortran compiler and other tools. Fortran-lang aims to provide a central place for Fortran users, complementing the language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.

Reviewer: ```suggestion
A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering, and in-spite of the evolution of the language itself, attracting new users and retaining existing ones is challenging. To address this, the open-source community Fortran-lang was formed in 2019. Here, we present the efforts of Fortran-lang including the Fortran package manager, standard library, website, LFortran compiler and other tools. Fortran-lang aims to provide a central home for all Fortran users, complementing the language standardization and compiler communities. Building on best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.
```
This is now 742 characters, with minor changes for style and consistency :)

Author: Thanks Rohit. I would not like to dwell too much over the use of past simple, past participle, etc. Indeed 2019 feels like long ago, but I think "has" in this context, serves to connect the action with the present.

I've taken Sebastian's suggestion to smoothen the use of _despite_ in the second sentence. I also kind of like "challenge for Fortran" at the stress position in the end. Let me know if the latest version looks good to you.

> A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering, and evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. To address this, an open-source community Fortran-lang has formed in 2019. Here, we present the efforts of Fortran-lang such as the Fortran package manager, standard library, website, LFortran compiler and other tools. Fortran-lang aims to provide a central place for Fortran users, complementing the language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.

Reviewer 2: Just some punctuation and rewording suggestions.

```suggestion
A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering and the evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. An open-source community Fortran-lang formed in 2019 to address this challenge. Here, we present the efforts of Fortran-lang: such as the Fortran package manager, standard library, website, LFortran compiler, and other tools. Fortran-lang aims to provide a central place for Fortran users, complementing language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.
```

Author: Thanks Sebastian. I don't think a colon is necessary after "such as". With respect to the third sentence, I would prefer to keep the current version which is 5 characters shorter, and states we are providing a solution at the beginning of the sentence. It also doesn't repeat "challenge" twice. Let me know if the latest version works for you.

Reviewer 3: 'An open-source community Fortran-lang': The title of the Discourse `Fortran Discourse - Fortran open source community` and all initiated projects are open source. But the community also includes developers and users of proprietary code (e.g. the Intel compiler).

> A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering and the evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. To address this, the open-source community Fortran-lang has formed in 2019. Here, we present the efforts of Fortran-lang such as the Fortran package manager, standard library, website, LFortran compiler, and other tools. Fortran-lang aims to provide a junction point for Fortran users, complementing the language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.

Reviewer 4: ```suggestion
A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering and the evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. To address this, the open-source community Fortran-lang has formed in 2019. We present the efforts of Fortran-lang including the package manager, standard library, website and LFortran compiler. Fortran-lang aims to provide a junction point for Fortran users, complementing the language standardization and compiler communities. We strive to reduce the learning curve for Fortran and promote its adoption by building on the best practices in software development.
```
From 751 to 729 characters.
The first altered sentence is just a bit shorter and simpler. I'm not sure about the changed structure of the last sentence, but I would like to replace 'aim' with 'strive' because 'aim' already appears in the sentence before.

Reviewer 5: Nice, but I think "other tools" had the very concrete role of representing the VScode plugin and the language server, so if there is space for it I'd suggest to keep it in. :)

Reviewer 4: By saying _we present the efforts including x, y, z_ it already implies there is more. I think "other tools" is just noise if you don't explicitly mention the VSCode and fortls.

Reviewer 5: Oh sorry didn't really parse the "including" instead of "such as", my bad. I agree that either way the reference is very generic so I guess both ways are fine.

Author: Thanks, I've adopted some of the changes. I would like to mention fortls and the VSCode extension.

# deRSE23 Submission

Abstract (and slides) for the [deRSE23](https://de-rse23.sciencesconf.org/) conference.

* Title: Fortran-Lang: an open source Fortran community
* Presenter: Ivan Pribec
* Co-authors: Vincent Magnin, Asdrubal Lozada-Blanco, Ondrej Certik, Jeremie Vandenplas

## Abstract

> A thriving community is vital for the success and continuity of a programming language. 
> Despite the importance of Fortran in science and engineering, and evolution of the language itself,
> attracting new users and retaining existing ones has proven challenging for Fortran.
> 
> In this talk, I will present the efforts of the Fortran-lang community including the Fortran package manager, standard library, website, LFortran compiler and other tools. 
> Fortran-lang aims to provide a central place for Fortran users,
> complementing the language standardization committee and compiler vendor communities.
> Building upon the common best practices in software development, 
> we aim to reduce the language learning curve and promote Fortran adoption.

Reviewer 6: Thank you for leading this effort. I think the abstract is in very good shape already. The only significant issue I found was that there was no single, to-the-point statement of a solution that immediately followed the problem statement (2nd sentence). I added the 3rd sentence to address this. Of course, to stay under 750 chars, I had to get creative in the remainder of the abstract and reword things a bit, without a change in content. I also removed the paragraph break--for an abstract this short, I think it reads best as a single paragraph. I hope you like the suggested changes, but I'm overall okay with the previous version as well.

```suggestion
A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering, and evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. To address this, an open-source community Fortran-lang has formed in 2019. Here, we present the efforts of Fortran-lang such as the Fortran package manager, standard library, website, LFortran compiler and other tools. Fortran-lang aims to provide a central place for Fortran users, complementing the language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.
```

Author: Thanks Milan! I agree with the problem-solution tension. In addition to the paragraph split you've shaved off a few characters by getting rid of the linebreaks 😄 .

# deRSE23 Submission

Abstract (and slides) for the [deRSE23](https://de-rse23.sciencesconf.org/) conference.

* Title: Fortran-Lang: an open source Fortran community
* Presenter: Ivan Pribec
* Co-authors: 

## Abstract

A thriving community is essential for the success and long-term survival of a programming language. 
Despite the importance of Fortran in science and engineering, and continuous developments in the language itself,
attracting new users and retaining existing ones has proven challenging for Fortran.

In this talk, I will share the motivations behind the new Fortran-lang community efforts. 

> This includes the Fortran package manager, standard library, website, and other tools. 

Reviewer 7: ```suggestion
This includes the Fortran package manager, standard library, website, LFortran compiler and other tools. 
```
